packages/manager/apps/cloud/client/app/dbaas/logs/detail/inputs/home/logs-inputs-home.html

```diff
@@ -4,28 +4,24 @@
 
     <div data-ng-if="ctrl.quota.loading"><oui-spinner></oui-spinner></div>
 
-    <p data-ng-if="!ctrl.quota.loading">
-        {{ctrl.quota.data.configured}}/{{ctrl.quota.data.max}}
-        <span data-translate="inputs_total_used"></span>
-        (<logs-upgrade-quota-link></logs-upgrade-quota-link>)
-    </p>
-
     <oui-spinner data-ng-if="ctrl.delete.loading"></oui-spinner>
     <oui-datagrid
         data-ng-if="!(ctrl.inputs.loading || ctrl.delete.loading)"
         rows="ctrl.inputs.data"
     >
         <extra-top>
-            <logs-add-tool
-                data-ng-if="!ctrl.quota.loading"
-                text="{{ ::'inputs_add_input' | translate }}"
-                tool-type="{{ ::'inputs_name' | translate }}"
-                current-usage="ctrl.quota.data.configured"
-                max-usage="ctrl.quota.data.max"
-                callback="ctrl.add()"
+            <button
+                type="button"
+                class="oui-button oui-button_secondary"
+                data-ng-click="ctrl.add()"
```

Comment:
loading

Reply:
Not sure to understand what you are mentioning here.
Could be please elaborate?
Thanks!